src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents the Contact data type in the address book. 
```

Comment:
No need to state where the class is used. (e.g. `in the address book.` is unnecessary)